Clarify PMTU management - #581 (#587)
quicwg/multipath: draft-ietf-quic-multipath.md

## Paths Having Different PMTU Sizes

An implementation should take care to handle different PMTU sizes across
multiple paths. PMTU size could even differ on the same 4-tuple given that

> multiple path can use the same 4-tuple.

Reviewer: ```suggestion
multiple paths can use the same 4-tuple.
```

An implementation should take care to handle different PMTU sizes across
multiple paths. As specified in {{Section 14.3 of QUIC-TRANSPORT}} the
DPLPMTUD Maximum Packet Size (MPS) is maintained for each combination of local and remote IP addresses.

> Note that with the multipath extension multiple paths could use the same 4-tuple
> which do not necessarily need to have the same MPS.

Reviewer: ```suggestion
Note that with the multipath extension multiple paths could use the same 4-tuple
but might have different MPS.
```

> One simple option, if the PMTUs are similar, is to apply the minimum PMTU of all paths to

Reviewer 2: This says the right thing, but does not sound right.

As specified in {{Section 14.3 of QUIC-TRANSPORT}} the
DPLPMTUD Maximum Packet Size (MPS) is maintained for each combination of local and remote IP addresses.
_However, the Internet architecture does not guarantee that the paths chosen for different path IDs will actually
experience the same MPS, even if the paths use the same 4-tuple. It might be more robust to independently
perform DPLPMTUD independently for each path._

Reviewer: That's okay for me but my main point was actually that other than with RFC9000, with the multipath extension you can have more than one path per 4-tuple/endpoint pair and therefore the statement to maintain one MSP per IP address pair is less useful.

Reviewer 2: OK. Maybe a simpler suggestion:

_Note that with the multipath extension multiple paths could use the same 4-tuple
but that these paths will not always have the same MPS._

Author: My original point was the same as @mirjak 's, but I also like the suggestion of independent DPLPMTUD.
Maybe, to make it a bit shorter, how about:
*However,  due to differences in routing, the paths chosen for different path IDs may not experience the same MPS, even if the paths use the same 4-tuple. It may be more robust to perform DPLPMTUD independently for each path.* ?

Author: Or even:
*Note that with the multipath extension multiple paths may use the same 4-tuple
but may have different MPS.* ?
